### PR TITLE
Meta: Move content to wiki

### PR DESCRIPTION
**Pull request by fregante**
I'm creating this PR to keep track of the changes required by the move of content to the wiki. @kidonng please include any further changes here

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2_feature_request.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ body:
   - type: textarea
     attributes:
       label: Description
-      description: 'Please ensure your requested feature makes sense for Refined GitHub: https://github.com/refined-github/refined-github/issues/2960'
+      description: 'Please ensure your requested feature makes sense for Refined GitHub: [info](https://github.com/refined-github/refined-github/wiki/%22Can-you-add-this-feature%3F%22)'
```
Comment (fregante) [inline .github/ISSUE_TEMPLATE/2_feature_request.yml:11]: word help plz

Comment (yakov116) [reply .github/ISSUE_TEMPLATE/2_feature_request.yml:11]:
```suggestion
      description: 'Please ensure your feature request falls into the [following guidelines](https://github.com/refined-github/refined-github/wiki/%22Can-you-add-this-feature%3F%22)'
```
Maybe?

Comment (yakov116) [reply .github/ISSUE_TEMPLATE/2_feature_request.yml]: Nice! I would linkify from the word `review`

**Comment by kidonng** [comment]
Merge https://github.com/refined-github/refined-github/wiki/GitHub-markdown-linkifier-bug into https://github.com/refined-github/refined-github/wiki/Extended-feature-descriptions? The former one just be a section about `prevent-link-loss`.

**Comment by fregante** [comment]
> Merge [refined-github/refined-github/wiki/GitHub-markdown-linkifier-bug](https://github.com/refined-github/refined-github/wiki/GitHub-markdown-linkifier-bug) into [refined-github/refined-github/wiki/Extended-feature-descriptions](https://github.com/refined-github/refined-github/wiki/Extended-feature-descriptions)? The former one can become a section about [`prevent-link-loss`](https://github.com/refined-github/refined-github/blob/main/source/features/prevent-link-loss.tsx).

Yeah I thought about making a single "extended feature descriptions" wiki, but got sidetracked/busy and eventually wasn't sure if this was a better idea after all. But you're free to try

**Pull request merged**